Generator/FamilyGenerator.php

```diff
@@ -85,27 +85,27 @@ public function generate(array $config, $outputDir, ProgressHelper $progress)
         $families = [];
 
         for ($i = 0; $i < $count; $i++) {
-            $family = [];
-            $family['labels']      = $this->getLocalizedRandomLabels();
+            $family           = [];
+            $family['labels'] = $this->getLocalizedRandomLabels();
 
             $family['attributeAsLabel'] = $this->labelAttribute;
 
             $attributes = $this->faker->randomElements($this->getAttributeCodes(), $attributesCount);
 
-            $attributes = array_merge([$this->identifierAttribute, $this->labelAttribute], $attributes);
+            $attributes           = array_merge([$this->identifierAttribute, $this->labelAttribute], $attributes);
```

Comment:
Nop, too many indentations. I agree it could be more readable to align "=" but only during init phase at the beginning of a method, not inside it.